### DISCUSSION TITLE
Улучшить поле ID

### DISCUSSION THREAD

**Discussion by MadZee-M** [moonshine-software/moonshine#929] [Ideas]
Доброго времени суток!

`ID::make('#')` - Тогда приходится добавить еще и второй аргумент `ID::make('#', 'id')` (не очень удобно). 
Т.е. сделать простое изменение имени для id без лишнего.

Проект отличный! Спасибо!

**Comment by lee-to** [comment]
ID::make()->setLabel('#')

**Comment by MadZee-M** [comment]
И еще один вопрос.

Почему не разрешаете использовать html в hint?

Мне нужно вывести снизу textarea некоторую информацию.

```
public function getHint(): string {
    return new HtmlString(parent::getHint());
}
```

Немного доработал этот метод под свои нужны, но на выводе получаю экранированный html.


> Посмотрел в исходниках. Можно решить изменив его на {!! ... !!}
> 
> ![2024-05-07_16-46](https://github.com/moonshine-software/moonshine/assets/32099597/65098a48-fe3b-4398-8316-c64f973b9cd1)

**Reply by lee-to** [reply]
Предложите pr
